src/components/graph/Graph.jsx

```diff
@@ -372,6 +372,23 @@ export default class Graph extends React.Component {
         this.props.onClickNode && this.props.onClickNode(clickedNodeId);
     };
 
+    /**
+     * Calls the callback passed to the component.
+     * @param  {Óbject} e - The event of onClick handler.
+     * @returns {undefined}
+     */
+    onClickGraph = e => {
```

Comment:
Please add this method to the component documentation, just follow the example of one of the functions there like `onClickNode`

Reply:
I just added it to Readme docs. Is there anywhere else you would like me to add it?

Reply:
Yes, please add it as well to the `jsdoc` that is on the component in the file `Graph.jsx`